Synapse/Api/Enum/LockerType.cs

```diff
@@ -2,11 +2,12 @@
 {
     public enum LockerType
     {
+        Default = -1,
```

Comment:
What for?

Reply:
https://github.com/SynapseSL/Synapse/pull/109/files#diff-3b00d15d5022fdc700a7c6dfb4c142403729c66c42ee54df438a1405f22b8a37R65

Reply:
Room.cs 65 line

Reply:
Locker*